### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pascalwengerter**
Preview can be found [here](https://github.com/owncloud/owncloud-sdk/tree/revamp-readme)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,29 @@
...
-<p align="center">
-  <img src="https://i.imgur.com/9mKra3O.png" />
-</p>
+# ownCloud JavaScript SDK
 
-Use this light-weight JS library with a promise-based interface for seamless communication with your ownCloud instance.<br>
-Supports only browser JS but requires JavaScript techniques like webpack to compile it properly.<br>
-For Node.js please have a look at [this tag](https://github.com/owncloud/owncloud-sdk/releases/tag/last-nodejs).
+Use this light-weight JS library with a promise-based interface for seamless communication with your ownCloud instance.
+Supports only browser JS but requires JavaScript techniques like webpack to compile it properly.
```
Comment (kulmann) [inline README.md:8]: Afaik it also supports node since we switched from sabredav to webdav client lib.

**Review by kulmann — COMMENTED** [review]
How is this failing in tests when only the readme is updated o_O

File: README.md (hunk 1 above)
Comment (pascalwengerter) [reply README.md]: Changed the wording to include Node as supported use case, CI perhaps had some hickup? 🤷🏽 it's green now

File: README.md [hunk 2]
```diff
@@ -58,25 +62,30 @@ oc.files.list('/path/to/file/folder').then(files => {
...
+### ownCloud file-picker
 
-Full API documentation is available at [![docs](https://img.shields.io/badge/api_docs-online-blue.svg)](https://owncloud.github.io/owncloud-sdk/)
+[ownCloud file-picker](github.com/owncloud/file-picker) is an integration to access the files in your ownCloud, e.g. in a chat app.
 
-## Building the Documentation
+## Documentation
 
-### JSDocs
+The full API documentation is available at [on the docs website](https://owncloud.dev/owncloud-sdk/).
```
Comment (phil-davis) [inline README.md:73]:
```suggestion
The full API documentation is available on [the docs website](https://owncloud.dev/owncloud-sdk/).
```

File: README.md [hunk 3]
```diff
@@ -58,25 +62,30 @@ oc.files.list('/path/to/file/folder').then(files => {
...
-$ yarn build:docs
+yarn build:docs
 ```
 
 ## Unit/Integration tests
+
 ### Overview
-owncloud-sdk uses [pactjs](https://github.com/pact-foundation/pact-js) with jest for unit and integration test.
+
+`owncloud-sdk` uses [pactjs](https://github.com/pact-foundation/pact-js) with jest for unit and integration test.
```
Comment (phil-davis) [inline README.md:88]:
```suggestion
`owncloud-sdk` uses [pactjs](https://github.com/pact-foundation/pact-js) with jest for unit and integration tests.
```

**Review by kulmann — APPROVED** [review]
Thanks for taking care of this 📝

**Pull request merged**